types/rsvp/index.d.ts

```diff
@@ -99,7 +99,8 @@ declare module 'rsvp' {
                 executor: (
                     resolve: (value?: RSVP.Arg<T>) => void,
                     reject: (reason?: any) => void
-                ) => void
+                ) => void,
+                label?: string
```

Comment:
Can you add a test for this?

Reply:
What would a test for this look like? Just invoking `new RSVP.Promise` with both arguments and ensuring there isn't a compilation error?

I see lots of assertions in the existing tests but am not sure how that would apply to the change I made here.

Reply:
yes, sounds good to me

Reply:
Cool, done.